### ISSUE TITLE
Outputting tons of JSON describing JSX elements 

### ISSUE THREAD

**Issue by davidtheclark** [6to5/6to5ify#26]
The transform is logging tons of JSON describing JSX elements. It looks like this:
```
{ type: 'XJSOpeningElement',
  start: 611,
  end: 640,
  loc: 
   { start: { line: 27, column: 3 },
     end: { line: 27, column: 32 } },
  range: [ 611, 640 ],
  attributes: 
   [ { type: 'Property',
       kind: 'init',
       key: [Object],
       value: [Object],
       computed: undefined,
       loc: [Object],
       end: 639,
       range: [Object],
       start: 619,
       leadingComments: [],
       trailingComments: [] } ],
  name: 
   { type: 'Identifier',
     start: 612,
     end: 618,
     loc: { start: [Object], end: [Object] },
     range: [ 612, 618 ],
     name: 'legend' },
  selfClosing: false }
{ type: 'XJSOpeningElement',
  start: 667,
  end: 716,
  loc: 
   { start: { line: 30, column: 4 },
     end: { line: 30, column: 53 } },
  range: [ 667, 716 ],
  attributes: 
   [ { type: 'Property',
       kind: 'init',
       key: [Object],
       value: [Object],
       computed: undefined,
       loc: [Object],
       end: 713,
       range: [Object],
       start: 681,
       leadingComments: [],
       trailingComments: [] } ],
  name: 
   { type: 'Identifier',
     start: 668,
     end: 680,
     loc: { start: [Object], end: [Object] },
     range: [ 668, 680 ],
     name: 'RequiredMark' },
  selfClosing: true }
{ type: 'XJSOpeningElement',
  start: 437,
  end: 484,
  loc: 
   { start: { line: 23, column: 3 },
     end: { line: 23, column: 50 } },
  range: [ 437, 484 ],
  attributes: 
   [ { type: 'Property',
       kind: 'init',
       key: [Object],
       value: [Object],
       computed: undefined,
       loc: [Object],
       end: 483,
       range: [Object],
       start: 443,
       leadingComments: [],
       trailingComments: [] } ],
  name: 
   { type: 'Identifier',
     start: 438,
     end: 442,
     loc: { start: [Object], end: [Object] },
     range: [ 438, 442 ],
     name: 'span' },
  selfClosing: false }
```

If I use browserify/6to5ify via the JS API, all this junk outputs to the console but the compiled code seems to work fine. However, if I use the browserify CLI and 6to5ify, the junk shows up *at the top of the compiled code*.

Any idea what is going on?

**Comment by sebmck** [comment]
Sorry! Looks like I accidently left a `console.log` in the last release... Fixed by #469 and released as of 2.11.1

**Comment by davidtheclark** [comment]
Glad it was an easy fix. Thanks!
